compose/src/main/java/com/bottlerocketstudios/compose/pullrequest/PullRequestEmptyScreen.kt

```diff
@@ -0,0 +1,40 @@
+package com.bottlerocketstudios.compose.pullrequest
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import com.bottlerocketstudios.compose.R
+import com.bottlerocketstudios.compose.resources.Colors
+import com.bottlerocketstudios.compose.util.Preview
+import com.bottlerocketstudios.compose.util.PreviewAll
+import com.bottlerocketstudios.launchpad.compose.bold
+
+@Composable
+fun PullRequestEmptyScreen() {
+    Column(
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxSize()
+    ) {
+        Text(
+            text = stringResource(R.string.no_pull_requests),
+            color = Colors.tertiary,
+            style = MaterialTheme.typography.h1.bold()
+        )
+    }
+}
+
+@PreviewAll
+@Composable
+fun HomeEmptyPreview() {
```

Comment:
@Br-DiorD nit: Make `private` and refactor name to `PullRequestEmptyScreenPreview`

```suggestion
private fun PullRequestEmptyScreenPreview() {
```